HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -993,6 +993,94 @@ def self.get_total_hvac_capacities(hpxml_bldg)
     return htg_cap, clg_cap, hp_backup_cap
   end
 
+  # Calculates total panel loads (across all panel loads for a given panel load type) for a given HPXML Building.
+  #
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @return [Array<Double * 15>] Total panel load for each panel load type (W)
+  def self.get_total_panel_loads(hpxml_bldg)
+    htg, clg, hw, cd, dw, ov, vf, sh, sp, ph, pp, wp, ev, ltg, oth = 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0
+    unit_multiplier = hpxml_bldg.building_construction.number_of_units
+    hpxml_bldg.electric_panels.each do |electric_panel|
+      htg += ElectricPanel.get_panel_load_heating(hpxml_bldg, electric_panel) * unit_multiplier
+      electric_panel.panel_loads.each do |panel_load|
+        if panel_load.type == HPXML::ElectricPanelLoadTypeCooling
+          clg += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeWaterHeater
+          hw += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeClothesDryer
+          cd += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeDishwasher
+          dw += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeRangeOven
+          ov += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeMechVent
+          vf += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypePermanentSpaHeater
+          sh += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypePermanentSpaPump
+          sp += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypePoolHeater
+          ph += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypePoolPump
+          pp += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeWellPump
+          wp += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeElectricVehicleCharging
+          ev += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeLighting
+          ltg += panel_load.power * unit_multiplier
+        elsif panel_load.type == HPXML::ElectricPanelLoadTypeOther
+          oth += panel_load.power * unit_multiplier
+        end
+      end
+    end
+    return htg, clg, hw, cd, dw, ov, vf, sh, sp, ph, pp, wp, ev, ltg, oth
+  end
+
+  # TODO
+  def self.get_total_breaker_spaces(hpxml_bldg)
```

Comment:
This is not meaningful, because each apartment will have their own unit panel that we must evaluate on independently. Unit panels are wired to the service main panel of the building along with other shared systems but I am not sure about the exact details.

Let's discuss.

Reply:
Perhaps load-based capacity shouldn't change based on the unit multiplier? Although for meter-based capacity, it would have to since peak electricity would already be scaled?